.gitignore

```diff
@@ -15,7 +15,7 @@ python_pylint_venv/
 htmlcov/
 dist/
 *.egg
-py27/
+py27*/
```

Comment:
Why this?

Reply:
There are now two virtualenviroments, py27_native and py27_gevent.

This makes sure they are both covered.

Reply:
Are we not doing any testing of gevent with Python 3?

Reply:
Ah, okay.

I have an awful feeling that `py3[0-9]*` was intended to be a regular expression rather than a glob. Knowing now that they are globs, can we rewrite these to only capture what we really intend to capture? It'd be nice in an explicitly self-documenting kind of way if these contained `_native` and `_gevent` in them.